feature-libs/asm/customer-360/root/model/customer-360.model.ts

```diff
@@ -42,6 +42,13 @@ export interface Customer360Coupon {
   name?: string;
 }
 
+export interface Customer360Promotion {
+  applied: boolean;
+  code?: string;
```

Comment:
code should be mandatory field I guess

Reply:
Yes, it's mandatory.I have corrected it.